AAUnlimited/Functions/Shared/Triggers/Expressions.cpp

```diff
@@ -2332,7 +2406,7 @@ namespace Shared {
 				{
 					40, EXPRCAT_EVENT,
 					TEXT("Npc Current Response Chance"), TEXT("CurrentResponsePercent"),
-					TEXT("If executed in a trigger with the Npc Answers Event, this is the current interaction percent, modified by this or previously executed triggers "
+					TEXT("If executed in a trigger with the Npc Answers Event, this is the current interaction percent, modified by this or previously executed triggers of normal priority. Will not work for super or absolute priority."
```

Comment:
"Will not work for super or absolute priority" -> "Will not work for strong or absolute responses"